external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -240,8 +240,8 @@ def _add_predictions_to_dataset(self, prediction_results, dataset, update_progre
             dataset_item.append_labels(item_labels)
 
             if feature_vector is not None:
-                active_score = TensorEntity(name="representation_vector", numpy=feature_vector)
-                dataset_item.append_metadata_item(active_score)
+                active_score = TensorEntity(name="representation_vector", numpy=feature_vector.reshape(-1))
```

Comment:
Is the `feature_vector` guaranteed to be already flat before reshaping. except for one or more axes of length one?
Would it be safer to `squeeze` instead, so that you don't accidentally flatten 3d vectors?

Reply:
They could be 3D in shape but actually 1D in semantic after spatial averaging. I think reshape(-1) or flatten() will be enough for standardizing feature vector.
@eugene123tw Could you confirm?

Reply:
Alright, if you have already averaged then any approach is fine.

Reply:
@eugene123tw yes, it's already averaged before so it's fine.